content/en/docs/setup/best-practices/enforcing-pod-security-standards.md

```diff
@@ -0,0 +1,36 @@
+---
+reviewers:
+- tallclair
+title: Enforcing Pod Security Standards
+weight: 40
+---
+
+<!-- overview -->
+
+This page provides an overview of best practices when it comes to enforcing [Pod Security Standards](/docs/concepts/security/pod-security-standards).
+
+<!-- body -->
+
+## Configure all cluster namespaces
```

Comment:
These best practices assume that you're using a mechanism based on namespace-level restrictions, but don't mention the built-in mechanism.

I think we should let readers know:
- there are 3rd party alternatives
- the PodSecurity admission mechanism is in alpha
- they could also use the (deprecated) PodSecurityPolicy feature

This page should help readers pick which approach to adopt given their circumstances. It's a tricky one because an alpha feature might not feel right for everyone, yet. We can explain that, provided readers are happy to handle the potential changes from minor release to minor release, the new admission controller is the best in-project mechanism.

Is there any generic advice about how to use or adopt 3rd party tools (OPA / Gatekeeper etc)? It might be as simple as “make sure you do _something_” or “make sure you trust your supply chain”.